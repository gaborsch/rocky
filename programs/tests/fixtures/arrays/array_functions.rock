Function takes the array
The product is 1
Until the array is nothing
Roll the array into the variable
Shout the variable
Let the product be the product of the variable

Give back the product

Rock the list with 3,4,5,6,6,8,9,10
Let the answer be function taking the list
Shout the answer (should be 1555200)

Shout the list (should be 8)

Let my heart be 12
Let my soul be 12
Rock my heart with my soul into your hands
Shout your hands (should be 1)
Shout your hands at 0 (should be 24)
