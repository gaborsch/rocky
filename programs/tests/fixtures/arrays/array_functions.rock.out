3
4
5
6
6
8
9
10
1555200
8
1
24
